packages/cannon-worker-shared/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@pmndrs/cannon-worker-shared",
```

Comment:
There should be 3 packages here:

1. `@pmndrs/cannon-worker-api`
2. `@react-three/cannon`
3. `@react-three/cannon-examples`

Reply:
I was thinking we could create a placeholder user-facing vanilla js cannon package `cannon-worker` as part of this.

But on second thought, maybe it's best to move the web worker api into `@pmndrs/cannon-worker-api` first, then look at that.

Renamed `@pmndrs/cannon-worker-shared` to `@pmndrs/cannon-worker-api` now